packages/terra-cli/index.js

```diff
@@ -0,0 +1,78 @@
+const yargs = require('yargs/yargs');
+const path = require('path');
+const fs = require('fs-extra');
+
+const Logger = require('./lib/utils/Logger');
+
+const setupCLI = () => {
+  const cli = yargs();
+
+  return cli.usage('Usage: $0 <command> [options]')
+    .demandCommand(1, 'A command is required. Pass --help to see all available commands and options.')
+    .recommendCommands()
+    .strict()
+    .fail((msg, err) => {
+      // certain yargs validations throw strings
+      const actual = err || new Error(msg);
+
+      // ValidationErrors are already logged, as are package errors
+      if (actual.name !== 'ValidationError' && !actual.pkg) {
+        // the recommendCommands() message is too terse
+        if (/Did you mean/.test(actual.message)) {
+          // eslint-disable-next-line no-console
+          console.error(`Unknown command "${cli.parsed.argv._[0]}"`);
+        }
+
+        // eslint-disable-next-line no-console
+        console.error(actual.message);
+      }
+
+      // exit non-zero so the CLI can be usefully chained
+      cli.exit(actual.code > 0 ? actual.code : 1, actual);
+    })
+    .alias('h', 'help')
+    .alias('v', 'version')
+    .wrap(cli.terminalWidth());
+};
+
+const subDirectoriesOfDirectory = (directoryPath) => (
+  fs.readdirSync(directoryPath, { withFileTypes: true })
+    .filter(directoryEntry => directoryEntry.isDirectory())
+    .map(directoryEntry => path.join(directoryPath, directoryEntry.name))
+);
+
+const terraClIDirectoriesFromPaths = (paths, preferSrc) => (
+  paths.map(dependencyPath => path.join(dependencyPath, preferSrc ? 'src' : 'lib', 'terra-cli')).filter(directory => fs.pathExistsSync(directory))
```

Comment:
why support both src and lib?

Reply:
This is to help out local development so you don't have to remember to compile to use the command.

Reply:
IMO we should setup local dev to run compile before running tests so it's not a problem so the CLI can be simplified